src/gatling/bodies/createResourceAccess.json

```diff
@@ -8,5 +8,6 @@
   "attributePermissions": {
     "": ["CREATE", "READ", "UPDATE"]
   },
+  "relationship": "Solicitor",
```

Comment:
Same has to be done for revoke method.

Reply:
Is solicitor added when application starts? If not would performance tests pass?

Clean your DB, start testing harness and double check that your performance tests and Postman collections work.

Reply:
👍